articles/communication-services/quickstarts/telephony-sms/includes/send-sms-python.md

```diff
@@ -63,7 +63,7 @@ except Exception as ex:
 While still in the application directory, install the Azure Communication Services SMS client library for Python package by using the `pip install` command.
 
 ```console
-pip install azure-communication-sms
+pip install azure-communication-sms --pre
```

Comment:
I had thought that we didn't need `--pre` anymore. Is this change because you noticed it was present in some other documentation? Can we check if the other SDKs like chat and auth have the --pre tag?

Reply:
I noticed the `--pre` tag is present in the SDK docs here
https://docs.microsoft.com/en-us/python/api/overview/azure/communication-sms-readme-pre?view=azure-python

Also, while trying to install this library using Pipenv, I get a locking error due to mismatch of sub dependencies.

Attaching the stacktrace of Pipenv that suggests using the `--pre` tag if it is a pre-release dependency
```
[ResolutionFailure]:       raise ResolutionFailure(message=str(e))
[pipenv.exceptions.ResolutionFailure]: Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  First try clearing your dependency cache with $ pipenv lock --clear, then try the original command again.
 Alternatively, you can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
  Hint: try $ pipenv lock --pre if it is a pre-release dependency.
ERROR: Could not find a version that matches azure-communication-sms (from -r /tmp/pipenv8qg6zqrprequirements/pipenv-6191vpx9-constraints.txt (line 2))
Skipped pre-versions: 1.0.0b1, 1.0.0b1, 1.0.0b2, 1.0.0b2, 1.0.0b3, 1.0.0b3, 1.0.0b4, 1.0.0b4
There are incompatible versions in the resolved dependencies:
```

Reply:
I see what you're saying, thanks for clarifying. Do you think we could also update the documentation here to be in sync? https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/telephony-sms/send?pivots=programming-language-python

Reply:
Yes.
Here is the PR
https://github.com/MicrosoftDocs/azure-docs/pull/67117